lib/components/form/call-taker/date-time-picker.tsx

```diff
@@ -187,6 +197,18 @@ const DateTimeOptions = ({
         })
       })
     }
+
+    if (
+      syncSortWithDepartArrive &&
+      DepartArriveTypeMap[departArrive as DepartArriveValue] !== sort.type
```

Comment:
cast `departArrive` wherever you get it from

Reply:
It's a `useState`. React doesn't like me typing it at definition time